app/src/main/java/co/nimblehq/compose/crypto/ui/screens/navigation/CryptoNavGraph.kt

```diff
@@ -0,0 +1,38 @@
+package co.nimblehq.compose.crypto.ui.screens.navigation
+
+import androidx.activity.ComponentActivity
+import androidx.compose.runtime.Composable
+import androidx.navigation.NavHostController
+import androidx.navigation.compose.NavHost
+import androidx.navigation.compose.composable
+import androidx.navigation.compose.rememberNavController
+import co.nimblehq.compose.crypto.extension.provideViewModels
+import co.nimblehq.compose.crypto.ui.screens.home.HomeScreen
+import co.nimblehq.compose.crypto.ui.screens.home.HomeViewModel
+import co.nimblehq.compose.crypto.ui.screens.information.CoinInfoScreen
+
+@Suppress("FunctionNaming")
+@Composable
+fun CryptoNavGraph(
+    componentActivity: ComponentActivity,
+    navController: NavHostController = rememberNavController(),
+    startDestination: String = CryptoScreen.HOME.name
+) {
+    NavHost(
+        navController = navController,
+        startDestination = startDestination
+    ) {
+        composable(CryptoScreen.HOME.name) {
```

Comment:
I think creating `CryptoScreen` as `enum` is unnecessary, I'd suggest creating `NavigationRoute`  and defined as the `const`. Actually, both are the same approach 😄 

https://github.com/nimblehq/jetpack-compose-ecommerce/blob/45ce5127940d7405dedae3d8d63b8c76f681b2cc/app/src/main/java/co/nimblehq/compose/ecommerce/navigation/Navigation.kt#L17-L31